atat_provisioning_wizard_api.yaml

```diff
@@ -961,73 +977,114 @@ components:
         task_orders:
           type: array
           items:
-            type: object
-            properties:
-              task_order_number:
-                type: string
-              task_order_file:
-                allOf:
-                  - $ref: '#/components/schemas/FileMetadataSummary'
-                description: Metadata associated with file which was previously uploaded by POST /taskOrderFiles
-              clins:
-                type: array
-                items:
-                  type: object
-                  properties:
-                    clin_number:
-                      type: string
-                    idiq_clin:
-                      type: string
-                    total_clin_value:
-                      type: "integer"
-                    obligated_funds:
-                      type: "integer"
-                    pop_start_date:
-                      type: string
-                      format: date
-                      example: '2021-07-01'
-                    pop_end_date:
-                      type: string
-                      format: date
-                      example: '2022-07-01'
+            $ref: '#/components/schemas/TaskOrderObject'
+    TaskOrderObject:
+      description: Represents a Task Order Object in the Funding Step
+      type: object
+      additionalProperties: false
+      properties:
+        task_order_number:
+          type: string
+        task_order_file:
+          description: Metadata associated with file which was previously uploaded by POST /taskOrderFiles
+          allOf:
+            - $ref: '#/components/schemas/FileMetadataSummary'
+        clins:
+          type: array
+          item:
+            $ref: '#/components/schemas/ClinObject'
+      required:
+        - task_order_number
+        - task_order_file
+        - clins
+    ClinObject:
+      description: Represents a Clin Object in the Funding Step
```

Comment:
Clin is within the Funding Step, but I think this would be better described as part of a Task Order